core/arch/x86/decode_table.c

```diff
@@ -3462,6 +3778,11 @@ const instr_info_t prefix_extensions[][8] = {
     {INVALID,    0xf30fd810, "(bad)", xx, xx, xx, xx, xx, no, x, END_LIST},
     {OP_vpsubusb, 0x660fd810, "vpsubusb", Vx, xx, Hx, Wx, xx, mrm|vex, x, END_LIST},
     {INVALID,    0xf20fd810, "(bad)", xx, xx, xx, xx, xx, no, x, END_LIST},
+    /* TODO i#1312: Support AVX-512. */
+    {INVALID,   0x0fd810, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID, 0xf30fd810, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID, 0x660fd810, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID, 0xf20fd810, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
```

Comment:
This is a big expansion of this table.  Will many of these be filled in?  If not, I would vote for instead making a separate table with 12 entries and only putting those that need an evex set there.

Reply:
Yes, I expect many or all of them to be filled in.